lib/runtime/src/lib.rs

```diff
@@ -152,7 +152,8 @@ pub fn instantiate(wasm: &[u8], import_object: &ImportObject) -> error::Result<I
     module.instantiate(import_object)
 }
 
-fn default_compiler() -> &'static dyn Compiler {
+/// Get a single instance of the default compiler to use.
+pub fn default_compiler() -> &'static dyn Compiler {
```

Comment:
Ideally, this shouldn't need to use the default compiler, but it that's easiest, it's okay.